next-docs/pages/world/systems.mdx

```diff
@@ -1 +1,111 @@
+
+When calling other contracts from a `System`, be aware that if you use `delegatecall` the called contract inherits the caller's permissions and can modify data from the `World` on behalf of the `System`.
+
+### Calling one `System` from another
+
+There are two ways to call one `System` from another one.
+
+| Call type                                    | `call` to the `World`                                                                                                                           | `delegatecall` directly to the `System`                                                                                                                         |
+| -------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Permissions                                  | those of the called `System`                                                                                                                    | those of the calling `System`                                                                                                                                   |
+| `_msgSender()`                               | calling `System` (unless you can use `callFrom`, which is only available when the user [delegates](/world/account-delegation) to your `System`) | can use [`WorldContextProvider`](https://github.com/latticexyz/mud/blob/main/packages/world/src/WorldContext.sol#L180-L203) to transfer the correct information |
+| `_msgValue()`                                | zero                                                                                                                                            | can use [`WorldContextProvider`](https://github.com/latticexyz/mud/blob/main/packages/world/src/WorldContext.sol#L180-L203) to transfer the correct information |
+| Can be used by systems in the root namespace | No (it's a security measure)                                                                                                                    | Yes                                                                                                                                                             |
+
+## Registering systems
+
+For a `System` to be callable from a `World` it has to be [registered](https://github.com/latticexyz/mud/blob/main/packages/world/src/modules/core/implementations/WorldRegistrationSystem.sol#L97-L201).
+If the namespace in which the `System` is registered already exists, only the [namespace owner](/world/namespaces-access-control#ownership).
+If it doesn't exist yet it is created, and the address that called `registerSystem` is assigned as the owner of the new namespace.
```

Comment:
Note that we're going to remove the auto-creation of namespaces shortly after the audit because it leads to frontrunning issues. My preference would be to already not mention it here.

```suggestion
Only the [namespace owner](./namespaces-access-control#ownership) can register a `System` in a namespace.
```